_includes/steering_group_election_section.html

```diff
@@ -6,6 +6,13 @@
 
     <h2>BIDS steering group election</h2>
 
+    <p>
+      This year's election has started, please check your BIDS email for a link to the ballot on <a href="https://opavate.com" target="_blank">opavote.com</a> and cast your vote.
```

Comment:
- Add one sentence to tell people to contact us if they believe they should have got an email and did not or update the sentence below that explains how to contact us.
```suggestion
      This year's election has started. 
      If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
      please check your BIDS email for a link to the ballot on <a href="https://opavate.com" target="_blank">opavote.com</a> and cast your vote.
```